lib/src/wallet/coin.dart

```diff
@@ -1,23 +1,29 @@
+import 'package:provenance_dart/src/wallet/private_key.dart';
+
 enum Coin {
   mainNet(
-    coinType: 505,
-    chainId: 'pio-mainnet-1',
-  ),
+      chainId: 'pio-mainnet-1',
+      defaultKeyPath: "m/44'/505'/0'/0/0",
+      prefix: "pb"),
   testNet(
-    coinType: 1,
-    chainId: 'pio-testnet-1',
-  );
+      chainId: 'pio-testnet-1',
+      defaultKeyPath: "m/44'/1'/0'/0/0'",
+      prefix: "tp");
 
   const Coin({
-    required this.coinType,
     required this.chainId,
+    required this.defaultKeyPath,
+    required this.prefix,
   });
 
-  final int coinType;
   final String chainId;
+  final String defaultKeyPath;
+  final String prefix;
 
-  static Coin forCoinType(int coinType) =>
-      values.firstWhere((e) => e.coinType == coinType);
+  static Coin forCoinType(int coinType) => values.firstWhere((e) {
```

Comment:
I don't think this is actually used anywhere if you prefer to remove it...

Reply:
if it is not being used then i will drop it